plugins/modules/apitoken.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import requests
+
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: apitoken
+
+short_description: Generate API Tokens
+
+version_added: "1.0.0"
+
+description: Generates API tokens given the OFFLINE token
+
+options:
+    offline_token:
+      description: The offline token used to generate the API Token
+      required: true
+
+author:
+    - Name (@vjayaramrh)
+"""
+
+EXAMPLES = r"""
+# Use argument
+- name: Generate API Token
+  apitoken
+    offline_token: blah
+
+"""
+
+RETURN = r"""
+[ "access_token", "expires_in", "id_token", "not-before-policy", "refresh_expires_in", "refresh_token", "scope", "session_state", "token_type" ]
```

Comment:
@tonyskapunk @clwheel How should the documentation be structured here? Thanks

Reply:
I would just include a redacted version of the dictionary that is returned, e.g.
```
{
  "access_token": "[redacted]",
  "expires_in": 900,
  "refresh_expires_in": 0,
  "refresh_token": "[redacted]",
  "token_type": "Bearer",
  "id_token": "[redacted]",
  "not-before-policy": 0,
  "session_state": "00000000-0000-0000-0000-000000000000",
  "scope": "openid api.iam.service_accounts roles web-origins offline_access"
}
```